terraform/README.md

```diff
@@ -0,0 +1,65 @@
+# Grafana-agent-k8s Terraform Module
+
+This Grafana-agent-k8s Terraform module aims to deploy the [grafana-agent-k8s charm](https://charmhub.io/grafana-agent-k8s) via Terraform.
+
+## Getting Started
+
+### Prerequisites
+
+The following software and tools needs to be installed and should be running in the local environment. Please [set up your environment](https://discourse.charmhub.io/t/set-up-your-development-environment-with-microk8s-for-juju-terraform-provider/13109) before deployment.
+
+- `microk8s`
+- `juju 3.x`
```

Comment:
Is Juju 3.x required? Terraform provider works with Juju `2.9.33+` and I don't think this charm requires Juju 3 specifically, does it?

Reply:
All of COS requires 3.x, except grafana-agent and cos-proxy.

Reply:
I will remove the version info as it could may vary depending on application type.

Reply:
Done